di_website/templates/events/event_index_page.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layouts/default.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block hero %}
+  {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+
+<section class="section">
+  <div class="row row--narrow">
+      <div class="l-sidebar">
+          <aside class="l-sidebar__aside">
+              <div class="aside aside--clipping">
+                  
+<h2 class="aside__heading">More in: What we do</h2>
+<nav class="navigation-secondary">
+<ul>
+
+<li class="navigation-secondary__item "><a href="#">Focus areas</a></li>
+
+<li class="navigation-secondary__item "><a href="#">Places we work</a></li>
+
+<li class="navigation-secondary__item "><a href="#">Consultancy services</a></li>
+
+<li class="navigation-secondary__item "><a href="#">Data support services</a></li>
+
+<li class="navigation-secondary__item "><a href="#">News</a></li>
+
+<li class="navigation-secondary__item navigation-secondary__item--active"><a href="#">Events</a></li>
+
+</ul>
+</nav>
+
+              </div>
+          </aside>
+          <div class="l-sidebar__body">
+              <div class="is-typeset wysiwyg">
```

Comment:
The formatting of this code sucks. Please fix it. VSCode can do auto-formating.

Reply:
Updated, thanks for the review